src/main/java/com/jvavateam/carsharingapp/dto/user/UserRequestDto.java

```diff
@@ -0,0 +1,13 @@
+package com.jvavateam.carsharingapp.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+public record UserRequestDto(@Email @NotBlank String email,
```

Comment:
I think that we can use that cross-field validation here, what if our password doesn't match repeated password?

Reply:
@holsoni Cross-field validation will implemented as an extra task. I have already added it to the Trello

Reply:
Sorry, maybe i missed that